README.md

```diff
@@ -55,16 +56,62 @@ $ python finetune.py --model_name pointconv \
     --epochs 50
 ```
 
-### finetuning model with unlabeled dataset
+#### finetuning model with unlabeled dataset
 
 ```bash
 $ python finetune.py --model_name pointconv \
     --train_dataset /path/to/unlabeled_data.bin \
     --interactive
 ```
 
+### Finetune pretrained-model with Pytorch Lightning
+#### prepare dataset
+
+To use your customized dataset, you should design your own dataset code, like those in `datasets/` directory. Here `datasets/modelnet40.py` is an example:
+
+```python
+class ModelNet40(torch.utils.data.Dataset):
+    def __init__(self, data_path, sample_points=1024, seed=10) -> None:
+        super().__init__()
+        # extract point data and labels from your file, e.g. npz, h5, etc.
+        data = np.load(data_path)
+        self.points = data['tensor']
+        self.labels = data['labels']
+        self.sample_points = sample_points
+
+    def __len__(self):
+        # return the total length of your data
+        return len(self.labels)
+
+    def __getitem__(self, index):
+        return (
+            # process on the fly, if needed
+            preprocess(self.points[index], num_points=self.sample_points),
+            self.labels[index],
+        )
+```
+
+#### finetuning model with labeled dataset
+
+```bash
+$ python finetune_pl.py --help
+
+$ python finetune_pl.py --model_name pointconv \
+    --train_dataset /path/to/train.bin \
+    --eval_dataset /path/to/eval.bin \
+    --split_ratio 0.8 \
+    --checkpoint_path /path/to/checkpoint/ \
+    --embed_dim 512 \
+    --hidden_dim 1024 \
+    --batch_size 128 \
+    --epochs 50
+```
 
```

Comment:
we need benchmark for each model finally

Reply:
ok, I will add it after curvenet finishes its training